### PR TITLE
Homepage and pricing

### PR DESCRIPTION
**Pull request by gastonsoto**
Home page and pricing page.

### CODE REVIEW

**Comment by martinlevesque** [comment]
Make sure both the top logo and logo at the bottom link to the homepage.

**Comment by martinlevesque** [comment]
Improve the section I changed "Build your side/fun projects, and growing company":

image

The list (ul) should be improved. If you have any ideas/suggestions regarding this section let me know. Note that many of our users are making side / fun projects, for example in spare times. It should be highlighted that it is optimized for this scenario.

**Comment by martinlevesque** [comment]
Add a new section "Open Source" in the homepage, maybe just before "Get started in minutes."

This section will contain few information about our open source initiative, which provide free hosting and highlight of "accepted" open source projects.
Then list about 3-4 featured open source projects

**Comment by martinlevesque** [comment]
In the pricing page, the top text (Get started now on opeNode! We provide a...) could be better aligned probably?

**Comment by martinlevesque** [comment]
In the pricing page, would it make sense to put the features (the same in the homepage) below the plans ?

**Comment by martinlevesque** [comment]
Menus hover/click improvements: the hover color in the top menu is not currently apparent.

Also when you click on an item, it shows some blue, not sure about this color (not used anywhere else?)

File: test/controllers/home_controller_test.rb [hunk 1]
```diff
@@ -5,76 +5,83 @@ class HomeControllerTest < ActionDispatch::IntegrationTest
     get '/'
 
     assert_response :success
-    assert_includes response.parsed_body, "Home#index"
+    assert_includes response.parsed_body, "| Home</title>"
```
Comment (martinlevesque) [inline test/controllers/home_controller_test.rb]: i think it's best to have the page specific title first, then more generic like opeNode:

<title>Home | opeNode</title>

**Comment by gastonsoto** [comment]
I believe I tackled all the comments. Please check them out.

**Comment by martinlevesque** [comment]
> In the pricing page, the top text (Get started now on opeNode! We provide a...) could be better aligned probably?

@gastonsoto would it be better if the text is not centered ? If you type a longer sentence it's taking probably too large ? Example:

image

This is something global to the site. If we edit/add text it would be more convenient if it looks good without having to manually add newlines.

File: app/views/home/pricing.html.erb [hunk 2]
```diff
@@ -1,2 +1,37 @@
-<h1>Home#pricing</h1>
-<p>Find me in app/views/home/pricing.html.erb</p>
+<section class="pricing-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+  <h1 class="display-4">Plans</h1>
+  <h2 class="text-muted">Get started now on opeNode!</h2>
+  <p class="lead">We provide a free sandbox to get started and you will not have to enter your credit card information.<br/>We accept both Paypal and Cryptocurrency (Bitcoin, Bitcoin Cash, etc.) payments.</p>
```
Comment (martinlevesque) [inline app/views/home/pricing.html.erb]: could be better that this part

> We provide a free sandbox to get started and you will not have to enter your credit card information.
We accept both Paypal and Cryptocurrency (Bitcoin, Bitcoin Cash, etc.) payments.

is left aligned (both We being aligned together), and also if the text is longer it should probably not take all space but change line automatically

File: app/views/home/pricing.html.erb [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+        <div class="card border-success shadow-sm ml-0 mr-0 h-100 hvr-grow">
+          <div class="card-header bg-success text-white">
+            <h4 class="my-0 font-weight-normal"><%= pricing[:name] %></h4>
+          </div>
+          <div class="card-body">
+            <h1 class="card-title pricing-card-title"><%= pricing[:title] %></h1>
+            <h3><small class="text-muted"><%= pricing[:subtitle] %></small></h3>
+            <ul class="list-unstyled mt-3 mb-4">
+              <% pricing[:features].each do |feature|  %>
+                <li><%= feature %></li>
```
Comment (martinlevesque) [inline app/views/home/pricing.html.erb]: would be better if features are aligned together (not centered) 

image

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -1,10 +1,296 @@
...
+      <div class="col-12 col-md-10 col-lg-8 text-center">      
+        <!-- Preheading -->
+        <h6 class="text-uppercase text-success font-weight-bold">
+          we support opensource
+        </h6>
+        <!-- Heading -->
+        <h1 class="font-weight-bold">Free Hosting For<br/><span class="text-secondary">Selected Projects</span></h1>
+        <div class="device-combo device-combo-macbook-iphonex mb-6 mb-md-0">
+          <div class="device device-macbook text-center">              
+            <img src="/images/undraw_open_source.svg" class="img-fluid" alt="...">
```
Comment (martinlevesque) [inline app/views/home/index.html.erb]: image

Comment (martinlevesque) [reply app/views/home/index.html.erb]: the background circle is not so apparent, so it creates a large white space area

File: app/views/home/index.html.erb [hunk 5]
```diff
@@ -1,10 +1,296 @@
...
+          </div>                          
+        </div>
+        <!-- Text -->
+        <div class="row text-gray-500 mt-5 mb-7 mb-md-9">
```
Comment (martinlevesque) [inline app/views/home/index.html.erb]: Add a title before the projects, something like:

image

**Comment by martinlevesque** [comment]
> I guess there is a typo. The **config/pricing.yml** file mentions the bandwidth of $0.40 as 10GB but it is currently 100GB.

@bil-ash hey thx you're right, this file is however only used to have some dev/test data. The real prices will be coming from an API call.

File: app/views/home/index.html.erb (hunk 4 above)
Comment (martinlevesque) [reply app/views/home/index.html.erb]: background color is now grey, not sure it fits with the layout

File: app/views/home/pricing.html.erb (hunk 3 above)
Comment (martinlevesque) [reply app/views/home/pricing.html.erb]: same issue @gastonsoto

File: app/views/home/pricing.html.erb [hunk 6]
```diff
@@ -1,2 +1,39 @@
-<h1>Home#pricing</h1>
-<p>Find me in app/views/home/pricing.html.erb</p>
+<section class="pricing-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto">
```
Comment (martinlevesque) [inline app/views/home/pricing.html.erb]: Does not display well on mobile, see

image

Comment (gastonsoto) [reply app/views/home/pricing.html.erb]: Fixed.

File: app/views/home/index.html.erb (hunk 4 above)
Comment (gastonsoto) [reply app/views/home/index.html.erb]: Fixed.

File: app/views/home/pricing.html.erb (hunk 3 above)
Comment (gastonsoto) [reply app/views/home/pricing.html.erb]: Added lines and more spacing between list elements.
Screen Shot 2019-12-30 at 15 22 54

Comment (martinlevesque) [reply app/views/home/pricing.html.erb]: looks good!

**Pull request merged**